ballerina/tests/test.bal

```diff
@@ -0,0 +1,284 @@
+// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+import ballerina/os;
+
+
+configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
+configurable string serviceUrl = isLiveServer ? os:getEnv("SERVICE_URL") : "http://localhost:9090";
+configurable string token = isLiveServer ? os:getEnv("JIRA_TOKEN") : "test";
+configurable string username = isLiveServer ? os:getEnv("JIRA_USERNAME") : "test@example.com";
+ConnectionConfig config = {
+    auth: <http:CredentialsConfig>
+    {
+        username: username,
+        password: token
+    }
+};
+final Client jira = check new Client(config, serviceUrl);
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetMyself() returns error? {
+    User response = check jira->/rest/api/'3/myself;
+
+    // Assertions
```

Comment:
By looking at the code, you can simply understand that we are doing an assertion. So shall we remove these comments as these do not add any value? Please check other places as well.